### PR TITLE
Readme vhs

### PR DESCRIPTION
**Pull request by forbesjo**
## Description
Updates the README to point to VHS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
+# Notice: The next version of this project can be found at [videojs-http-streaming](https://github.com/videojs/http-streaming)
```
Comment (gesinger) [inline README.md]: It may be worth pointing to the VJS 7 blog post or have some short description of the reason why it's the next version.

**Comment by forbesjo** [comment]
- [ ] Add a warning for npm install to point to VHS

File: README.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 [![Slack Status][slack-icon]][slack-link]
 [![Greenkeeper badge][greenkeeper-icon]][greenkeeper-link]
 
-
 Play back HLS with video.js, even where it's not natively supported.
 
 Lead Maintainer: Jon-Carlos Rivera [@imbcmdth](https://github.com/imbcmdth)
 
-Maintenance Status: Stable
+Maintenance Status: Deprecated
```
Comment (gesinger) [inline README.md]: We may want to wait to update this line until it's officially deprecated.

Comment (gkatsev) [reply README.md]: we can leave this as deprecated but then archive the repo when it's actually deprecated

**Pull request merged**